### PR TITLE
chapters: Review contents of assignment section

### PR DESCRIPTION
**Pull request by razvand**
@Alex-deVis , see my comments, update the PR, you can push to my branch.

### CODE REVIEW

**Review by Alex-deVis — COMMENTED** [review]
I like the proposed structure, but I need some clarification before getting to work.

File: chapters/use-deliver/assignments/reading/README.md [hunk 1]
```diff
@@ -4,12 +4,40 @@ Owner: Alex Apostolescu
 Reviewer: Sergiu Weisz
 
 Assignments are unassisted practical activities that learners undertake individually or in teams over a longer timespan.
-Their primary objective is to provide learners the flexibility to work through tasks at their desired speed, allowing them to address gaps in their understanding as they progress.
+Their primary objective is to provide learners the flexibility to work through tasks at their preferred speed, allowing them to address gaps in their understanding as they progress.
+
+RD: Assignment attributes:
```
[the remark this answers is not in the document]
Comment (razvand) [reply chapters/use-deliver/assignments/reading/README.md:9]: It's up to you. Decide what it should look like.

File: chapters/use-deliver/assignments/reading/README.md [hunk 2]
```diff
@@ -4,12 +4,40 @@ Owner: Alex Apostolescu
...
+
+- contents: statement, skeleton, testing, solution
+- timeline
```
[the remark this answers is not in the document]
Comment (razvand) [reply chapters/use-deliver/assignments/reading/README.md:12]: Agree, drop it.

File: chapters/use-deliver/assignments/reading/README.md [hunk 3]
```diff
@@ -4,12 +4,40 @@ Owner: Alex Apostolescu
...
+- grading
+- support channel
+- submission interface
+
+Summary of steps:
```
[the remark this answers is not in the document]
Comment (razvand) [reply chapters/use-deliver/assignments/reading/README.md:17]: You decide. But it's required to provide the reader an overview of what they would get.

File: chapters/use-deliver/assignments/reading/README.md [hunk 4]
```diff
@@ -4,12 +4,40 @@ Owner: Alex Apostolescu
...
+#### RD: Preparing an Assignment
+
+- selecting contents: drills (questions and / or tasks) and / or projects
+- creating an assignment on the implementation repository: linking above items from the content repository
+- preparing the validation infrastructure
+
+I would only add preparing an assignment and grading an assignment as prime sections.
+These are related to use of content.
+
+The others (Launching Assingments and Providing Support) fall under "Others", they are not a relevant part of this methodology.
```
[the remark this answers is not in the document]
Comment (razvand) [reply chapters/use-deliver/assignments/reading/README.md]: @Alex-deVis , agree.

File: chapters/use-deliver/assignments/reading/README.md [hunk 5]
```diff
@@ -1,31 +1,45 @@
...
-In the initial stages of the assignment, most queries pertain to administrative and workflow-related aspects.
+However, if you decide to create a project, there are several fundamental components to ensure, such as the assignment statement, support files, and the reference solution.
+These steps are detailed in the ["projects" section](../../../develop-organize/projects/reading/README.md).
+
+The last steps involve setting up the validation infrastructure, which also includes the creation of the submission form, and establishing channels for assistance.
+These steps are strongly tied to the subject's infrastructure.
+
+#### Launching Assignments
+
+The initial phase of launching an assignment involves making everyting prepared (assignment files, submission forms, assistance channels) public.
```
Comment (razvand) [inline chapters/use-deliver/assignments/reading/README.md:27]:
```suggestion
The initial phase of launching an assignment involves making sure everything is properly set (assignment files, submission forms, assistance channels) public.
```

File: chapters/use-deliver/assignments/reading/README.md [hunk 6]
```diff
@@ -91,3 +95,13 @@ Yet, the latter approach could potentially lead to a significantly heavier workl
 
 To address this concern, the utilization of tools like [Google Docs](https://www.google.com/docs/about/) proves invaluable.
 These tools offer version history and the ability to highlight changes made between different revisions, significantly enhancing the overall quality and efficiency of the grading process.
+
+The last step of the grading phase is the plagiarism check.
+This important step prevents students from copying solutions from each other or external sources to ensure a fair academic environment.
+To ensure objectivity, this step should incorporate an automated check.
+
+To address this concern, available public anti-plagiarism tools can be used.
+Whenever feasible, it's advisable to opt for specialized tools, such as [MOSS](https://theory.stanford.edu/~aiken/moss/) for code-centric assignments.
```
Comment (razvand) [inline chapters/use-deliver/assignments/reading/README.md:104]: Add [JPlag](https://github.com/jplag/JPlag).

**Review by razvand — COMMENTED** [review]
This looks good. However, you require a section on setting up the validation infrastructure, or at least referencing it in the infrastructure section. Otherwise, it seems "it just works".

**Comment by Alex-deVis** [comment]
@razvand, I applied your feedback, this is ready for merge. Waiting for @Sergiu121's review.

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**